lbry/lbry/extras/daemon/Daemon.py

```diff
@@ -375,6 +375,13 @@ def installation_id(self):
                 install_id_file.write(self._installation_id)
         return self._installation_id
 
+    @staticmethod
+    def log_function_params(fname, **params):
+        params_passed = [f'{k}: {repr(v)}' for k, v in params.items() if v is not None]
+        logstr = f'{fname}: {{{", ".join(params_passed)}}}'
+        log.info(logstr)
```

Comment:
Does the default log formatting system not work for this?

Reply:
No, only if either an error occurs, or a log is specified directly.